vkcs/networking/resource_port.go

```diff
@@ -95,16 +95,26 @@ func ResourceNetworkingPort() *schema.Resource {
 				ForceNew:    false,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				Set:         schema.HashString,
-				Description: "(Conflicts with `no_security_groups`) A list of security group IDs to apply to the port. The security groups must be specified by ID and not name (as opposed to how they are configured with the Compute Instance).",
+				Computed:    true,
+				Description: "(Conflicts with `no_security_groups`) A list of security group IDs to apply to the port. The security groups must be specified by ID and not name. External changes are not displayed by default, to see them you need to set the `full_security_groups_control` argument to true.",
```

Comment:
A list of security group IDs to apply to the port. The security groups must be specified by ID and not name. If the list is empty no one security group is attached to the port. If the argument is absent `vkcs_networking_port` resource does not control security groups of the port just reads them. _note_ This behavior is actual with `full_security_groups_control` = true only and introduced in X.Y.Z version of the provider. Legacy behavior is still supported but deprecated and too confusing to be described.

Reply:
done